diff --git a/src/plugins/robots/pi-puck/simulator/dynamics3d_pipuck_model.cpp b/src/plugins/robots/pi-puck/simulator/dynamics3d_pipuck_model.cpp
index e0ffe7a..13ec9b8 100644
--- a/src/plugins/robots/pi-puck/simulator/dynamics3d_pipuck_model.cpp
+++ b/src/plugins/robots/pi-puck/simulator/dynamics3d_pipuck_model.cpp
@@ -190,10 +190,12 @@ namespace argos {
    /****************************************/
    /****************************************/
 
-   const btVector3    CDynamics3DPiPuckModel::m_cBodyHalfExtents(0.0362, 0.069, 0.0362);
+   /*
+   //const btVector3    CDynamics3DPiPuckModel::m_cBodyHalfExtents(0.0362, 0.069, 0.0362);
+   const btVector3    CDynamics3DPiPuckModel::m_cBodyHalfExtents(0.1000, 0.039, 0.1000);
    const btScalar     CDynamics3DPiPuckModel::m_fBodyMass(0.001);
    const btTransform  CDynamics3DPiPuckModel::m_cBodyOffset(btQuaternion(0.0, 0.0, 0.0, 1.0), btVector3(0.0,0.00125,0.0));
-   const btTransform  CDynamics3DPiPuckModel::m_cBodyGeometricOffset(btQuaternion(0.0, 0.0, 0.0, 1.0), btVector3(0.0, -0.069, 0.0));
+   const btTransform  CDynamics3DPiPuckModel::m_cBodyGeometricOffset(btQuaternion(0.0, 0.0, 0.0, 1.0), btVector3(0.0, -0.039, 0.0));
    const btVector3    CDynamics3DPiPuckModel::m_cWheelHalfExtents(0.02125, 0.0055, 0.02125);
    const btScalar     CDynamics3DPiPuckModel::m_fWheelMass(0.1265);
    const btTransform  CDynamics3DPiPuckModel::m_cWheelGeometricOffset(btQuaternion(0.0, 0.0, 0.0, 1.0), btVector3(0.0, -0.0015, 0.0));
@@ -205,14 +207,44 @@ namespace argos {
    const btVector3    CDynamics3DPiPuckModel::m_cBodyToLeftWheelJointOffset(0.0, -0.049, -0.0255);
    const btVector3    CDynamics3DPiPuckModel::m_cLeftWheelToBodyJointOffset(0.0, 0.0015, -0.0);
    const btQuaternion CDynamics3DPiPuckModel::m_cBodyToLeftWheelJointRotation(btVector3(1,0,0), SIMD_HALF_PI);
-   /* TODO calibrate these values */
    const btScalar     CDynamics3DPiPuckModel::m_fBodyFriction(0.0);
    const btScalar     CDynamics3DPiPuckModel::m_fWheelMotorMaxImpulse(0.05);
    const btScalar     CDynamics3DPiPuckModel::m_fWheelFriction(10);
    const btScalar     CDynamics3DPiPuckModel::m_fWheelMotorClamp(5.0);
    const btScalar     CDynamics3DPiPuckModel::m_fWheelMotorKdCoefficient(2.0);
+   */
 
+   //const btVector3    CDynamics3DPiPuckModel::m_cBodyHalfExtents(0.0362, 0.069, 0.0362);
+   const btVector3    CDynamics3DPiPuckModel::m_cBodyHalfExtents(0.1000, 0.0375, 0.1000);
+   const btScalar     CDynamics3DPiPuckModel::m_fBodyMass(0.001);
+   const btTransform  CDynamics3DPiPuckModel::m_cBodyOffset(btQuaternion(0.0, 0.0, 0.0, 1.0), btVector3(0.0,0.00125,0.0));
+   //const btTransform  CDynamics3DPiPuckModel::m_cBodyGeometricOffset(btQuaternion(0.0, 0.0, 0.0, 1.0), btVector3(0.0, -0.069, 0.0));
+   const btTransform  CDynamics3DPiPuckModel::m_cBodyGeometricOffset(btQuaternion(0.0, 0.0, 0.0, 1.0), btVector3(0.0, -0.0375, 0.0));
+   //const btVector3    CDynamics3DPiPuckModel::m_cWheelHalfExtents(0.02125, 0.0055, 0.02125);
+   const btVector3    CDynamics3DPiPuckModel::m_cWheelHalfExtents(0.03125, 0.0055, 0.03125);
+   const btScalar     CDynamics3DPiPuckModel::m_fWheelMass(0.1265);
+   //const btTransform  CDynamics3DPiPuckModel::m_cWheelGeometricOffset(btQuaternion(0.0, 0.0, 0.0, 1.0), btVector3(0.0, -0.0015, 0.0));
+   const btTransform  CDynamics3DPiPuckModel::m_cWheelGeometricOffset(btQuaternion(0.0, 0.0, 0.0, 1.0), btVector3(0.0, -0.0055, 0.0));
+   //const btTransform  CDynamics3DPiPuckModel::m_cLeftWheelOffset(btQuaternion(btVector3(-1,0,0), SIMD_HALF_PI), btVector3(0.0, 0.02125, -0.0255));
+   //const btTransform  CDynamics3DPiPuckModel::m_cRightWheelOffset(btQuaternion(btVector3(1,0,0), SIMD_HALF_PI), btVector3(0.0, 0.02125, 0.0255));
+   const btTransform  CDynamics3DPiPuckModel::m_cLeftWheelOffset(btQuaternion(btVector3(-1,0,0), SIMD_HALF_PI), btVector3(0.0, 0.03125, -0.0255));
+   const btTransform  CDynamics3DPiPuckModel::m_cRightWheelOffset(btQuaternion(btVector3(1,0,0), SIMD_HALF_PI), btVector3(0.0, 0.03125, 0.0255));
+   //const btVector3    CDynamics3DPiPuckModel::m_cBodyToRightWheelJointOffset(0.0, -0.049, 0.0255);
+   //const btVector3    CDynamics3DPiPuckModel::m_cBodyToRightWheelJointOffset(0.0, -0.0175, 0.0255);
+   const btVector3    CDynamics3DPiPuckModel::m_cBodyToRightWheelJointOffset(0.0, -0.0075, 0.0255);
+   const btVector3    CDynamics3DPiPuckModel::m_cRightWheelToBodyJointOffset(0.0, 0.0055, 0.0);
+   const btQuaternion CDynamics3DPiPuckModel::m_cBodyToRightWheelJointRotation(btVector3(-1,0,0), SIMD_HALF_PI);
+   //const btVector3    CDynamics3DPiPuckModel::m_cBodyToLeftWheelJointOffset(0.0, -0.049, -0.0255);
+   //const btVector3    CDynamics3DPiPuckModel::m_cBodyToLeftWheelJointOffset(0.0, -0.0175, -0.0255);
+   const btVector3    CDynamics3DPiPuckModel::m_cBodyToLeftWheelJointOffset(0.0, -0.0075, -0.0255);
+   const btVector3    CDynamics3DPiPuckModel::m_cLeftWheelToBodyJointOffset(0.0, 0.0055, -0.0);
+   const btQuaternion CDynamics3DPiPuckModel::m_cBodyToLeftWheelJointRotation(btVector3(1,0,0), SIMD_HALF_PI);
+   /* TODO calibrate these values */
+   const btScalar     CDynamics3DPiPuckModel::m_fBodyFriction(0.0);
+   const btScalar     CDynamics3DPiPuckModel::m_fWheelMotorMaxImpulse(0.05);
+   const btScalar     CDynamics3DPiPuckModel::m_fWheelFriction(10);
+   const btScalar     CDynamics3DPiPuckModel::m_fWheelMotorClamp(5.0);
+   const btScalar     CDynamics3DPiPuckModel::m_fWheelMotorKdCoefficient(2.0);
    /****************************************/
    /****************************************/
-
 }
diff --git a/src/plugins/robots/pi-puck/simulator/qtopengl_pipuck.cpp b/src/plugins/robots/pi-puck/simulator/qtopengl_pipuck.cpp
index b9fa8fe..a1d5ef8 100644
--- a/src/plugins/robots/pi-puck/simulator/qtopengl_pipuck.cpp
+++ b/src/plugins/robots/pi-puck/simulator/qtopengl_pipuck.cpp
@@ -325,8 +325,17 @@ namespace argos {
    /****************************************/
    /****************************************/
 
-   const CVector3 CQTOpenGLPiPuckDebug::m_cBodyScaling                          {0.0724, 0.0724, 0.0472};
-   const CVector3 CQTOpenGLPiPuckDebug::m_cWheelScaling                         {0.0425, 0.0425, 0.003};
+   /*
+   //const CVector3 CQTOpenGLPiPuckDebug::m_cBodyScaling                          {0.0724, 0.0724, 0.0472};
+   const CVector3 CQTOpenGLPiPuckDebug::m_cBodyScaling                          {0.2000, 0.2000, 0.078};
+   const CVector3 CQTOpenGLPiPuckDebug::m_cWheelScaling                         {0.0425, 0.0425, 0.011};
+   */
+
+   //const CVector3 CQTOpenGLPiPuckDebug::m_cBodyScaling                          {0.0724, 0.0724, 0.0472};
+   const CVector3 CQTOpenGLPiPuckDebug::m_cBodyScaling                          {0.2000, 0.2000, 0.0750};
+   //const CVector3 CQTOpenGLPiPuckDebug::m_cWheelScaling                         {0.0425, 0.0425, 0.003};
+   //const CVector3 CQTOpenGLPiPuckDebug::m_cWheelScaling                         {0.0425, 0.0425, 0.011};
+   const CVector3 CQTOpenGLPiPuckDebug::m_cWheelScaling                         {0.0625, 0.0625, 0.001};
 
    const std::array<GLfloat, 4> CQTOpenGLPiPuckDebug::m_arrDefaultColor         {0.5f, 0.5f, 0.5f, 1.0f};
    const std::array<GLfloat, 4> CQTOpenGLPiPuckDebug::m_arrDefaultSpecular      {0.0f, 0.0f, 0.0f, 1.0f};
