diff --git a/src/plugins/robots/drone/simulator/pointmass3d_drone_model.cpp b/src/plugins/robots/drone/simulator/pointmass3d_drone_model.cpp
index 42af876..aeea463 100644
--- a/src/plugins/robots/drone/simulator/pointmass3d_drone_model.cpp
+++ b/src/plugins/robots/drone/simulator/pointmass3d_drone_model.cpp
@@ -423,18 +423,18 @@ namespace argos
    const Real CPointMass3DDroneModel::MEMS_GYRO_BIAS_INIT = 0.0007;
    const Real CPointMass3DDroneModel::MEMS_GYRO_RANDOM_WALK_INIT = 0.0007;
    const Real CPointMass3DDroneModel::MEMS_GYRO_NOISE_MEAN = 0;
-   const Real CPointMass3DDroneModel::MEMS_GYRO_NOISE_STD_DEV_X = 0.25;
-   const Real CPointMass3DDroneModel::MEMS_GYRO_NOISE_STD_DEV_Y = 0.25;
-   const Real CPointMass3DDroneModel::MEMS_GYRO_NOISE_STD_DEV_Z = 0.1;
+   const Real CPointMass3DDroneModel::MEMS_GYRO_NOISE_STD_DEV_X = 0.75;
+   const Real CPointMass3DDroneModel::MEMS_GYRO_NOISE_STD_DEV_Y = 0.75;
+   const Real CPointMass3DDroneModel::MEMS_GYRO_NOISE_STD_DEV_Z = 0.3;
    /* Accelerometer sensor noise coefficents */
    const Real CPointMass3DDroneModel::MEMS_ACCEL_BIAS_STD_DEV = 0.002;
    const Real CPointMass3DDroneModel::MEMS_ACCEL_BIAS_MEAN = 0;
    const Real CPointMass3DDroneModel::MEMS_ACCEL_BIAS_INIT = 0.0009;
    const Real CPointMass3DDroneModel::MEMS_ACCEL_RANDOM_WALK_INIT = 0.0009;
    const Real CPointMass3DDroneModel::MEMS_ACCEL_NOISE_MEAN = 0;
-   const Real CPointMass3DDroneModel::MEMS_ACCEL_NOISE_STD_DEV_X =  0.3;
-   const Real CPointMass3DDroneModel::MEMS_ACCEL_NOISE_STD_DEV_Y = 0.3;
-   const Real CPointMass3DDroneModel::MEMS_ACCEL_NOISE_STD_DEV_Z = 0.3;
+   const Real CPointMass3DDroneModel::MEMS_ACCEL_NOISE_STD_DEV_X =  1.0;
+   const Real CPointMass3DDroneModel::MEMS_ACCEL_NOISE_STD_DEV_Y = 1.0;
+   const Real CPointMass3DDroneModel::MEMS_ACCEL_NOISE_STD_DEV_Z = 1.0;
 #endif
 
    /****************************************/
